app-libs/stf/Cargo.toml

```diff
@@ -18,6 +18,9 @@ sgx_tstd = { branch = "master", features = ["untrusted_fs", "net", "backtrace"],
 # local crates
 ita-sgx-runtime = { default-features = false, path = "../sgx-runtime" }
 itp-hashing = { default-features = false, path = "../../core-primitives/hashing" }
+itp-node-api = { path = "../../core-primitives/node-api", default-features = false }
+itp-node-api-metadata = { path = "../../core-primitives/node-api/metadata", default-features = false }
+itp-node-api-metadata-provider = { path = "../../core-primitives/node-api/metadata-provider", default-features = false }
```

Comment:
I apologize for being nitpicky here, but we usually have the `default-features = false` as the first entry, so that the important things are first. :)

Reply:
No problem, I adjusted it